lstchain/tools/lstchain_create_sampling_interval_coefficient_file.py

```diff
@@ -0,0 +1,262 @@
+import numpy as np
+from astropy.io import fits
+import h5py
+from tqdm import tqdm
+
+from ctapipe.core import Provenance, traits
+from ctapipe.core import Tool
+from ctapipe_io_lst import LSTEventSource
+from ctapipe_io_lst.calibration import LSTR0Corrections
+from lstchain.calib.camera.sampling_interval_coefficient_calculate import SamplingIntervalCalculate
+from lstchain.paths import run_info_from_filename
+
+class SamplingIntervalCoefficientHDFWriter(Tool):
+    """
+    Tool that generates a sampling interval coefficient hdf file.
+    For getting help run:
+    lstchain_create_sampling_interval_coefficient_file --help
+    """
+
+    name = "SamplingIntervalCoefficientHDFWriter"
```

Comment:
Hi Seiya,
if I understand correctly you are writing fits files, instead of a HDF5 file.  
Would be it possible to write indeed a h5 file? (we decided to prefer this format)

Reply:
Forget my comment, I see that the final file is in h5 and also that you put together the gains at the ends. 
Perhaps a bit misleading the fact that you have single tool for the three steps